.storybook/themes/news.scss

```diff
@@ -912,3 +943,4 @@
 @use "../../blocks/story-carousel-block";
 @use "../../blocks/triple-chain-block";
 @use "../../blocks/video-player-block";
+@use "../../blocks/links-bar-block";
```

Comment:
can you add the links bar to the list based on alphabetical order. it's not said in the docs but I use the sort ascending tool on vs code